detox/src/devices/android/ADB.js

```diff
@@ -9,7 +10,16 @@ const Environment = require('../../utils/environment');
 class ADB {
   constructor() {
     this._cachedApiLevels = new Map();
-    this.adbBin = path.join(Environment.getAndroidSDKPath(), 'platform-tools', 'adb');
+    try{
+      this.adbBin = path.join(Environment.getAndroidSDKPath(), 'platform-tools', 'adb');
+    } catch (err){
+      const adbOnPath =  which.sync('adb', {nothrow: true});
+      if (adbOnPath) {
+        this.adbBin = adbOnPath;
+        return;
+      }
+      throw new Error(err);
```

Comment:
So, if you care that much about rethrowing the original error, can you show please how does this error look like? Is it comprehensible enough for a human?